.github/actions/aws_marketplace_ecr_login/action.yml

```diff
@@ -0,0 +1,16 @@
+name: 'AWS Marketplace | AWS ECR Login'
```

Comment:
For consistency, in the dir name, let's use dashes instead of underscores.

- `.github/actions/aws_marketplace_ecr_login` => `.github/actions/aws-marketplace-ecr-login`
- - `.github/actions/aws_marketplace_ecr_mirror` => `.github/actions/aws-marketplace-ecr-mirror`

Reply:
I renamed this